# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/procps/hotfix-signames.patch.sparc
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- procps-ng-4.0.5/library/signames.c.vanilla	2025-07-24 16:51:09.626692793 +0200
+++ procps-ng-4.0.5/library/signames.c	2025-07-24 16:51:37.430386319 +0200
@@ -100,9 +100,9 @@
         SIGABBREV(PWR);
 #endif
         SIGABBREV(SYS);
-#ifdef SIGLOST              // Hurd (gitlab#93)
-        SIGABBREV(LOST);
-#endif
+//#ifdef SIGLOST              // Hurd (gitlab#93)
+//        SIGABBREV(LOST);
+//#endif
 #if defined __sun || defined __SUN || defined __solaris__ || defined __SOLARIS__
         case 0: return "EXIT";
 #endif
